src/query/time_of_impact_internal/shape_against_shape.rs

```diff
@@ -35,3 +35,36 @@ pub fn shape_against_shape<N: RealField>(
         panic!("No algorithm known to compute a contact point between the given pair of shapes.")
     }
 }
+
+/// Computes the contact normal and the smallest time of impact of two shapes
+/// under translational movement.
+///
+/// The returned time of impact will be `0.0` if the objects are touching or
+/// penetrating.
```

Comment:
It's better to have a convention regarding the returned normal. So we can document it:
```suggestion
/// penetrating.
/// The returned normal points toward the exterior of the first shape at the impact point.
```